tests/Console/ConfigurationResolverTest.php

```diff
@@ -961,6 +961,26 @@ public function provideResolveRulesCases()
                     '@A' => true,
                 ),
             ),
+            '@A - @C' => array(
```

Comment:
Analyzing the issue more calmly today I think there was a bit of a misunderstanding.

We don't have a `@Foo -@Bar` anywhere. What we have is a `@Foo, -@Bar` (notice the comma). Which is the command line way of saying `Use @Foo but disable @Bar`. Which is equivalent to `{'@Foo': true, '@Bar': false}` on a json or a 
```
return PhpCsFixer\Config::create()
    ->setRules(array(
        '@Foo' => true,
        '@Bar' => false
    )
    ->setFinder(
        PhpCsFixer\Finder::create()
            ->exclude('tests/Fixtures')
            ->in(__DIR__)
    );
```

So there is no operation between sets, just plain old **enable this**, **disable that**. I couldn't find any Rule Set on **RuleSet.php** with a `'rule' => false` either. Allowing a Rule Set to forcibly disable a specific rule seems a bit unnecessary and can lead to this sort of confusion we're having, imho.

@SpacePossum, then a `--rules=@SF, @CommentFixers` would  behave as you intend and there would be no such thing as a `--rules=@SF minus @CommentFixers`. That is considering an hypothetical Rule Set that has a `'header' => false` arbitrarily.

What do you guys think?